### PR TITLE
Added some packages to repos file. Updated versions

### PR DESCRIPTION
**Pull request by jsantiago-eProsima**
Added python bindings and micro-ros-agent. Updated versions to their latest releases.

Signed-off-by: Javier Santiago <javiersantiago@eprosima.com>

### CODE REVIEW

File: vulcanexus.repos [hunk 1]
```diff
@@ -1,53 +1,57 @@
...
+    version: c607bd1564fd39311a574a509e7f3c53367b4c44
+  eProsima/shapes-demo:
+    type: git
+    url: https://github.com/eProsima/ShapesDemo.git
+    version: v2.6.0
+  eProsima/vulcanexus_base:
+    type: git
+    url: https://github.com/eProsima/vulcanexus.git
+    version: be93dd7906dfef09dc10fe75376ddee60c3e001e
+  ros2/rmw_fastrtps:
```
Comment (JLBuenoLopez-eProsima) [inline vulcanexus.repos:50]: Are we sure that we want to have this package inside the `ros2` folder? It is not the same RMW as the one provided by ROS 2...

File: vulcanexus.repos [hunk 2]
```diff
@@ -1,53 +1,57 @@
...
+    type: git
+    url: https://github.com/eProsima/rmw_fastrtps.git
+    version: 6f93e84db42076809f3e3adb7aed467a02d725ad
+  ros2/rosidl_typesupport_fastrtps:
+    type: git
+    url: https://github.com/ros2/rosidl_typesupport_fastrtps.git
+    version: 1.2.1
```
Comment (JLBuenoLopez-eProsima) [inline vulcanexus.repos:57]:
```suggestion
    version: 2.1.0
```

Do we want to support latest release or the same release that is supported in ROS 2 Galactic?

File: vulcanexus.repos [hunk 3]
```diff
@@ -1,53 +1,57 @@
...
+    url: https://github.com/eProsima/Fast-DDS-Gen.git
+    version: v2.1.2
+  eProsima/foonathan_memory_vendor:
+    type: git
+    url: https://github.com/eProsima/foonathan_memory_vendor.git
+    version: v1.2.1
+  eProsima/micro_ros_agent:
+    type: git
+    url: https://github.com/micro-ROS/micro-ROS-Agent.git
+    version: 2.0.1
```
[the remark this answers is not in the document]
Comment (jsantiago-eProsima) [reply vulcanexus.repos:37]: Micro-ROS uses the major version number to identify the ROS distribution that branch targets. 2.x.x tags are the correct ones then using Galactic.

File: vulcanexus.repos [hunk 4]
```diff
@@ -1,53 +1,57 @@
...
+    url: https://github.com/micro-ROS/micro_ros_setup.git
+    version: c607bd1564fd39311a574a509e7f3c53367b4c44
+  eProsima/shapes-demo:
+    type: git
+    url: https://github.com/eProsima/ShapesDemo.git
+    version: v2.6.0
+  eProsima/vulcanexus_base:
+    type: git
+    url: https://github.com/eProsima/vulcanexus.git
+    version: be93dd7906dfef09dc10fe75376ddee60c3e001e
```
[the remark this answers is not in the document]
Comment (jsantiago-eProsima) [reply vulcanexus.repos:49]: These hashes provide better traceability. Simply using galactic (or vulcanexus-galactic) does not suffice in the event we need to reference the correct source at a certain point in time should any issue require it.

File: vulcanexus.repos [hunk 5]
```diff
@@ -1,53 +1,57 @@
...
+  ros2/rmw_fastrtps:
+    type: git
+    url: https://github.com/eProsima/rmw_fastrtps.git
+    version: 6f93e84db42076809f3e3adb7aed467a02d725ad
```
[the remark this answers is not in the document]
Comment (jsantiago-eProsima) [reply vulcanexus.repos:53]: Same here.

File: vulcanexus.repos (hunk 2 above)
Comment (jsantiago-eProsima) [reply vulcanexus.repos:57]: 1.2.1 is the latest Galactic version, so it's the one we are going to use.

File: vulcanexus.repos (hunk 1 above)
Comment (jsantiago-eProsima) [reply vulcanexus.repos:50]: Moved it to eProsima folder.

File: vulcanexus.repos [hunk 6]
```diff
@@ -1,53 +1,57 @@
...
-        version: galactic
-    eProsima/dds_router:
-        type: git
-        url: https://github.com/eProsima/DDS-Router.git
-        version: v0.1.0
-    eProsima/vulcanexus_base:
-        type: git
-        url: https://github.com/eProsima/vulcanexus.git
-        version: galactic
+  eProsima/Fast-CDR:
```
[the remark this answers is not in the document]
Comment (jsantiago-eProsima) [reply vulcanexus.repos:2]: Homogenized all package names to be similar to `eProsima/Fast-CDR`

File: vulcanexus.repos [hunk 7]
```diff
@@ -1,53 +1,57 @@
...
+    url: https://github.com/eProsima/foonathan_memory_vendor.git
+    version: v1.2.1
+  eProsima/micro_ros_agent:
+    type: git
+    url: https://github.com/micro-ROS/micro-ROS-Agent.git
+    version: 2.0.1
+  eProsima/micro_ros_setup:
+    type: git
+    url: https://github.com/micro-ROS/micro_ros_setup.git
+    version: c607bd1564fd39311a574a509e7f3c53367b4c44
```
[the remark this answers is not in the document]
Comment (jsantiago-eProsima) [reply vulcanexus.repos]: Added tag 2.0.0

File: vulcanexus.repos (hunk 4 above)
[the remark this answers is not in the document]
Comment (JLBuenoLopez-eProsima) [reply vulcanexus.repos:53]: Yes, we would probably follow the same philosophy as ROS, having two branches, the `galactic-release` branch that will have a specific commit/tag and the `galactic` branch which will have the associated `galactic` development branches of the packages.

**Comment by jsantiago-eProsima** [comment]
Regarding [this comment](https://github.com/eProsima/vulcanexus/pull/13#discussion_r836491314) this is exactly the plan. A "Galactic Rolling" branch, and the tagged release one.

**Pull request merged**